ads/google/a4a/utils.js

```diff
@@ -148,6 +167,9 @@ export function googleAdUrl(
         {name: 'brdim', value: additionalDimensions(win, viewportSize)},
         {name: 'isw', value: viewportSize.width},
         {name: 'ish', value: viewportSize.height},
+        {name: 'ea', value: '0'},
```

Comment:
We should make sure to look into expandable support.  Technically you can resize in AMP but not sure it works with what expandable would expect.  For now just add a TODO.